### ISSUE TITLE
[TIL] 김현재 201106

### ISSUE THREAD

**Issue by journey101** [codestates/ds-TIL#836]
- **키워드**:  
> ML예측모델(lightGBM, catboost, randomforest, XGBoost)
> 검증지표(f1, AUC, accuracy), recall, precision 

- **배운 것**: 
> 머신러닝 활용의 다양한 사례와 분석법, 인사이트 도출 사례 

- **더 알고 싶은 부분 / 공부하고 싶은 부분**: 
> 데이터를 취합 가공하는 부분에 대해서 더 다양한 데이터들로 연습해봐야겠다는 생각. 

- **느낀 점**: 
> 실제 해보는 것이 정말 큰 배움임을 느낍니다. 
모두 고생하셨습니다. 감사합니다.
